smoketest/relay/go.mod

```diff
@@ -45,4 +46,4 @@ require (
 	google.golang.org/protobuf v1.34.2 // indirect
 )
 
-replace github.com/TBD54566975/ftl => /Users/safeer/dev/ftl
+replace github.com/TBD54566975/ftl => ../..
```

Comment:
This was the main change/fix for this PR

Reply:
😬 thanks Wes